--- glob-old.c	2019-08-08 23:34:27.761483106 -0700
+++ glob.c	2018-12-03 17:53:31.998013794 -0800
@@ -93,6 +93,8 @@
 #include <wchar.h>
 
 #include "collate.h"
+#include "glob.h"
+#include "freebsd-compat.h"
 
 /*
  * glob(3) expansion limits. Stop the expansion if any of these limits
@@ -863,7 +865,7 @@
 
 	newn = 2 + pglob->gl_pathc + pglob->gl_offs;
 	/* reallocarray(NULL, newn, size) is equivalent to malloc(newn*size). */
-	pathv = reallocarray(pglob->gl_pathv, newn, sizeof(*pathv));
+	pathv = malloc(sizeof(*pathv));
 	if (pathv == NULL)
 		return (GLOB_NOSPACE);
 
